Entities/Items/Containers/Chest/Chest.as

```diff
@@ -137,7 +140,7 @@ void onDie(CBlob@ this)
 		1.0f,                               // scale?
 		0,                                  // ?
 		"",                                 // sound
-		this.get_u8("team_color"));         // team number
+		this.get_u8("color"));              // team number
```

Comment:
I feel like if you could just replace this with `this.getNetworkID() % 5` would remove the need for storing "color"

Reply:
But then the chest won't have a randomized color. Chests would have a predetermined color in TDM maps.